### DISCUSSION TITLE
UMW sensors aren't compatible with each other?

### DISCUSSION THREAD

**Discussion by zachliu** [cpetrich/counterfeit_DS18B20#44] [General]
I have this UMW SOP-8 DS18B20 on a pHAT board with external power, the 4.7k pull-up resistor and everything. This chip is likely from DigiKey (not 100% sure though).

![SOP-8](https://github.com/user-attachments/assets/46b1b32c-c8de-40a5-bfaf-3190a9ff990a)

I also bought five TO-92 DS18B20A10 from UMW through their direct distribution channel.

![ds18b20a10](https://github.com/user-attachments/assets/2005189e-0bb3-4eec-a1d6-1e37896797e8)

Funny thing is whenever I connect the five TO-92 sensors, the onboard one would disappear from the device list. 

![2024-10-31_09-59_1](https://github.com/user-attachments/assets/e004337c-7469-47ce-895e-06b120a5be00)
(almost got consecutive serial numbers here)

But once I remove the TO-92s, the onboard SOP-8 would re-appear :joy: 

![2024-10-31_09-59](https://github.com/user-attachments/assets/72c57f8f-7cbc-4223-841b-9d173df33da6)

Any ideas?

**Comment by cpetrich** [comment]
Well, that's somewhat entertaining. I know nothing about SOP-8 chips or the pHAT, I'm afraid. Back in 2019 I had sensors of Family D1 reproducibly disappear when I connected too many of them at once, but never ran into problems with B1 and B2 series sensors. No idea other than I'll try to reproduce this with TO-92s one of these days.

**Reply by zachliu** [reply]
i'm starting to doubt the authenticity of that SOP-8 chip because i can't trace its exact purchase origin.
also, is the `0223` from its id `28-022367277cff`  representing manufacture date? if so, it doesn't match with the chip marking `38th week of 2023` :thinking:
